+dj/+internal/Settings.m

```diff
@@ -58,6 +59,13 @@
             if nargin == 2
                 new_state = subsasgn(current_state, subscript, value);
                 stateAccess('set', new_state);
+                if ~isempty(subscript(1).subs) && strcmp(subscript(1).subs, 'use32BitDims')
```

Comment:
Is this first check necessary? :thinking: 

Wondering if we are safe since we check above that `ischar(name)` already.

```suggestion
                if strcmp(subscript(1).subs, 'use32BitDims')
```